sdcm/cluster.py

```diff
@@ -2448,8 +2448,11 @@ def prepare_files_for_archive(self, fileslist):
                 self.remoter.run('cp -r %s %s' % (f, new_full_path), ignore_status=True)
         return root_dir
 
-    def generate_coredump_file(self, restart_scylla=True):
-        self.log.info('Generate scylla core')
+    def generate_coredump_file(self, restart_scylla=True, node_name: str = None, message: str = None):
+        message = message or "Generating a Scylla core dump by SIGQUIT..\n"
+        if node_name:
+            message += f"Node: {node_name}"
+        InfoEvent(severity=Severity.ERROR, message=message)
         self.remoter.run("sudo pkill -f --signal 3 /usr/bin/scylla")
```

Comment:
if you are already changing, please us `self.remoter.sudo("pkill -f --signal 3 /usr/bin/scylla")` instead

Reply:
fixed